docs/source/implementations/portfolio_optimisation.rst

```diff
@@ -6,6 +6,9 @@ Portfolio Optimisation
 
 The portfolio optimisation module contains different algorithms that are used for asset allocation and optimising strategies. Each algorithm is encapsulated in its own class and has a public method called ``allocate()`` which calculates the weight allocations on the specific user data. This way, each implementation can be called in the same way and makes it simple for users to use them. Next up, lets discuss about some of these implementations and the different parameters they require.
```

Comment:
There is a whitespace in the beggining here.

Reply:
Resolved